ironic/drivers/modules/drac/raid.py

```diff
@@ -134,6 +134,27 @@ def list_physical_disks(node):
         raise exception.DracOperationError(error=exc)
 
 
+def is_raid_controller(node, raid_controller_fqdd, raid_controllers=None):
```

Comment:
Can you add an entry for the raid_controllers param in the comments below?

Reply:
Added